mysqld_exporter.go

```diff
@@ -52,6 +63,56 @@ func NewMySQLExporter(dsn string) *Exporter {
 			Help:      "The last scrape error status.",
 		}),
 		metrics: map[string]prometheus.Gauge{},
+		commands: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: namespace,
+			Name:      "commands_total",
+			Help:      "Number of executed mysql commands.",
+		}, []string{"command"}),
+		connection_errors: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: namespace,
+			Name:      "connection_errors_total",
+			Help:      "Number of mysql connection errors.",
+		}, []string{"error"}),
+		handlers: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: namespace,
+			Name:      "handlers_total",
+			Help:      "Number of executed mysql handlers.",
+		}, []string{"handler"}),
+		innodb_buffer_pool: prometheus.NewUntypedVec(prometheus.UntypedOpts{
+			Namespace: namespace,
+			Name:      "innodb_buffer_pool_total",
+			Help:      "Mysql Innodb buffer pool metrics.",
+		}, []string{"metric"}),
+		innodb_data: prometheus.NewUntypedVec(prometheus.UntypedOpts{
+			Namespace: namespace,
+			Name:      "innodb_data_total",
+			Help:      "Mysql Innodb data operations.",
+		}, []string{"operation"}),
+		innodb_pages: prometheus.NewUntypedVec(prometheus.UntypedOpts{
+			Namespace: namespace,
+			Name:      "innodb_pages_total",
+			Help:      "Mysql Innodb page operations.",
+		}, []string{"operation"}),
+		innodb_rows: prometheus.NewUntypedVec(prometheus.UntypedOpts{
+			Namespace: namespace,
+			Name:      "innodb_rows_total",
+			Help:      "Mysql Innodb row operations.",
+		}, []string{"operation"}),
+		performance_schema: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: namespace,
+			Name:      "performance_schema_total",
```

Comment:
This isn't on https://dev.mysql.com/doc/refman/5.1/en/server-status-variables.html and I suspect there's a better way to export it.